tests/acceptance/stepDefinitions/notificationsContext.js

```diff
@@ -0,0 +1,24 @@
+const { client } = require('nightwatch-api')
+const { When, Then } = require('cucumber')
+const httpHelper = require('../helpers/httpHelper')
+const fetch = require('node-fetch')
+
+When('user {string} is sent a notification', function (user) {
+  return fetch(
+    client.globals.backend_url + '/ocs/v2.php/apps/testing/api/v1/notifications',
+    { method: 'POST', headers: httpHelper.createAuthHeader(user), body: { 'user': user } }
+  )
+    .then(res => httpHelper.checkStatus(res, 'Could not generate notification.'))
+})
+
+When('the user marks the notification as read', function () {
+  return client.page.phoenixPage().clickOnMarkAsRead()
+})
+
+Then('the user should see the notification bell on the webUI', function () {
+  return client.page.phoenixPage().checkForNotificationBellVisible()
```

Comment:
you should be able to use `expect` here https://nightwatchjs.org/api/#expect-visible
checks / assertions should be usually done in the context